realtime-image-recognition/src/main/java/RealTimeImageRecognition.java

```diff
@@ -105,6 +105,15 @@ private static Pipeline buildPipeline(String modelPath) {
     }
 
     public static void main(String[] args) {
+        //TODO https://github.com/hazelcast/hazelcast-jet-demos/issues/88
```

Comment:
Can you please extract this logic into a method to not pollute the main method ?

Reply:
Done